### DISCUSSION TITLE
PWA's on Linux / OSX

### DISCUSSION THREAD

**Discussion by 32bitx64bit** [Floorp-Projects/Floorp#675] [Feature Request]
Would love to see PWA support on Linux / OSX (MacOS)

**Comment by p134c0d3** [comment]
I did bring this up, and Mozilla would need to provide the API in order for this to happen. They provided the API for Windows, but have not provided the API for Linux. It sucks. Would have to take it up with Mozilla I think. I'm trying to figure out where to go to do that honestly.

**Reply by 32bitx64bit** [reply]
Ah typical Mozilla. Hopefully something gets worked out
